woocommerce/woocommerce
# Add active theme label to intro screen

### Changes proposed in this Pull Request:

Closes #40794.

This PR adds the check for current theme in the theme list in CYS intro screen. Likely a temporary solution since theme list is still hardcoded.



### How to test the changes in this Pull Request:

1. Make sure to enable `customize-store` feature flag
2. If your environment is not hosted in WPCOM, download this fake `tsubaki` theme [[tsubaki.zip](https://github.com/woocommerce/woocommerce/files/12928740/tsubaki.zip)]. 
3. Install & activate `Tsubaki` theme
1. Go to `WooCommerce -> Home` and click `Customize Store` task
4. Observe the current active theme is highlighted

<img width="806" alt="image" src="https://github.com/woocommerce/woocommerce/assets/3747241/c136f543-cf2c-4c19-89d7-8173af4e1ad8">


### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
